codes/extractive/netsum/netsum.py

```diff
@@ -185,6 +196,48 @@ def data_prep(text_path, summary_path, news_words_path, is_lemmatize, file_info,
             
     return X, Y
 
+def get_sent_word_sens(sentence):
+    '''
+    Get senses for each valid word in a sentence.
+    
+    Parameter:
+        sentence (str): a string containing a collection of words.
+        
+    Return:
+        wordSense (list): a list of extracted sense (default to be the first one)
+    '''
+    wordSense = []
+    for w in re.findall(r'[a-zA-Z]+', sentence):
+        if w in STOP_WORDS or len(w) < WORD_MIN_LEN:
+            continue
+        senses = wn.synsets(w)
```

Comment:
I tried to add pos='v' here and it kinds of hurt the performance. So we can do this for now.

Reply:
Great.